frontend/src/app/modules/members/members-page/members-page.component.ts

```diff
@@ -178,6 +178,17 @@ export class MembersPageComponent extends BaseComponent implements OnInit {
             });
     }
 
+    approve(member: Member) {
+        this.memberService.approveMember(member.id)
+            .pipe(this.untilThis)
+            .subscribe(() => {
+                this.toastNotifications.success('Member approved');
+                window.location.reload();
```

Comment:
Why do we need this line?

Reply:
This need to reload the page to get new data

Reply:
Please don't use reloading the page becouse it is SPA.

Reply:
Fix